Makefile.automated

```diff
@@ -442,7 +463,110 @@ elwiki_models: \
 
 elwiki_tuning_reports: \
 	tuning_reports/elwiki.reverted.md
+
+############################# English Wikipedia ################################
+
+datasets/enwiki.human_labeled_revisions.20k_2015.json:
+	./utility fetch_labels \
+		https://labels.wmflabs.org/campaigns/enwiki/4/ > $@
+
+datasets/enwiki.labeled_revisions.20k_2015.json: \
+		datasets/enwiki.human_labeled_revisions.20k_2015.json
+	./utility merge_labels $^ > $@
+
+datasets/enwiki.autolabeled_revisions.w_cache.20k_2015.json: \
+		datasets/enwiki.autolabeled_revisions.20k_2015.json
```

Comment:
Missing a rule to create enwiki.autolabeled_revisions.20k_2015.json

Reply:
Removed the whole thing